### PR TITLE
Cu-1w9414y unit tests for asset handler

### PR DESCRIPTION
**Pull request by rkamysz**
PR includes:

- unit tests for the asset handler
- assets fixtures
- mocked fastify instance

### CODE REVIEW

File: src/api-handlers/__tests__/asset.unit.test.ts [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    });
+
+    expect(collection).toEqual([]);
+  });
+
+  it('Should throw an error when no document with the given "id" was found', async () => {
+    const findResult = [];
+    const collection = await getAssetsCollection(getAssetsFastifyMock({ findResult }), {
+      query: {
+        id: '61f7f245a92817601180e8eb',
```
Comment (montera82) [inline src/api-handlers/__tests__/asset.unit.test.ts:82]: nitpick; if we put fakeowner0 in a variable in line71

**Comment by montera82** [comment]
good job generally.LGTM aside from merge conflict

**Comment by rkamysz** [comment]
solved

**Comment by rkamysz** [comment]
Unfortunately I also pushed `printWidth` lint option which caused some changes in files not related to assets. Lets decide on printWidth value 90, 100, 120 and format whole code in other PR. sorry again

**Comment by dskvr** [comment]
Task linked: [CU-1w9414y Write unit tests for asset handler](https://app.clickup.com/t/1w9414y)

**Review by montera82 — APPROVED** [review]

**Pull request merged**